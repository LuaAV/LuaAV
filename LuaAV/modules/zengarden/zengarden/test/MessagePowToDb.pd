#N canvas 696 193 450 300 10;
#X obj 104 17 loadbang;
#X obj 104 183 print;
#X obj 173 17 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X msg 139 99 -14.3;
#X msg 104 99 3.7;
#X msg 197 99 0;
#X obj 104 54 t b b b b;
#X msg 235 99 24 35 48;
#X obj 104 148 powtodb;
#X connect 0 0 6 0;
#X connect 2 0 6 0;
#X connect 3 0 8 0;
#X connect 4 0 8 0;
#X connect 5 0 8 0;
#X connect 6 0 4 0;
#X connect 6 1 3 0;
#X connect 6 2 5 0;
#X connect 6 3 7 0;
#X connect 7 0 8 0;
#X connect 8 0 1 0;
