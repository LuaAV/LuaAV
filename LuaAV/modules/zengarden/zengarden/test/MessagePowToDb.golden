[@ 0.000ms] print: 113.802
[@ 0.000ms] print: 0
[@ 0.000ms] print: 0
[@ 0.000ms] print: 105.682